### ISSUE TITLE
model_name required

### ISSUE THREAD

**Issue by rajeeja** [ECP-CANDLE/Benchmarks#114]
With the updates in candle_lib `model_name` is a required hyperparameter. Breaks UNO and might other Benchmarks also.

**Comment by jmohdyusof** [comment]
Yes, I have updated all the benchmarks to comply. Did you find an error? First line of uno_default_model.txt is model_name

**Comment by rajeeja** [comment]
Gotcha. Not all .txt files were changed. I was using `uno_auc_model.txt`  in my fork. https://github.com/rajeeja/Benchmarks/tree/AUCstd1

**Comment by jmohdyusof** [comment]
OK, I basically just swept all the xxx_default_model.txt files, I would have missed any secondary .txt files (most benchmarks do not have more than the default_model). I can sweep the others. We also need to fix the path issue

**Comment by rajeeja** [comment]
Is `--config_file ` broken?

**Comment by jmohdyusof** [comment]
It is because Tom wanted non-default files to be in data_dir. I am working with Cristina to fix it
